security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,13 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-2-1"]
+== OpenShift File Integrity Operator 1.2.1
+
+The following advisory is available for the OpenShift File Integrity Operator 1.2.1:
```

Comment:
For releases where we're not shipping any new FIO code, but just updating the container dependencies, do we typically call that out in the release notes?

Reply:
We have not done it in the past, like [FIO 1.0.0](https://docs.openshift.com/container-platform/4.12/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-1-0-0). But we could add a line below that says "This release includes updated container dependencies." ?